traffic_ops/traffic_ops_golang/routes.go

```diff
@@ -103,6 +104,8 @@ func Routes(d ServerData) ([]Route, []RawRoute, http.Handler, error) {
 		{1.1, http.MethodPost, `asns/?$`, api.CreateHandler(asn.GetTypeSingleton()), auth.PrivLevelOperations, Authenticated, nil},
 		{1.1, http.MethodDelete, `asns/{id}$`, api.DeleteHandler(asn.GetTypeSingleton()), auth.PrivLevelOperations, Authenticated, nil},
 
+		{1.1, http.MethodGet, `caches/stats/?(\.json)?$`, cachesstats.Get, auth.PrivLevelReadOnly, Authenticated, nil},
```

Comment:
I don't think this is unique to this endpoint, but this will allow me to `GET` `/api/1.x/caches/stats/.json`. I checked and it seems that the Perl allows this, so maybe for that reason alone we don't want to change it, but it's weird.